practice_app/practice_app/urls.py

```diff
@@ -14,8 +14,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', include("search.urls"))
```

Comment:
I am not sure about this line and I want to get your idea also. I believe it seems cleaner to me when the URLs divided into apps. What I mean is path('/search', include("search.urls")).

But you also implemented homepage; therefore, the path of the homepage also must be changed. I don't know if it is a good idea or not, we can discuss in the comments.

Reply:
During the implementation, I created a home page and used it in various ways. It was necessary for the development process. 
But, in the deployment process, it is nice to have separate URLs as you said. Therefore, I changed it and made like path('search/', include("search.urls")).

Reply:
Thanks!